flow/designs/asap7/ibex/rules.json

```diff
@@ -12,42 +12,42 @@
         },
         {
             "field": "placeopt__design__instance__design__area",
-            "value": 3146,
+            "value": 3119,
             "compare": "<="
         },
         {
             "field": "placeopt__design__instance__stdcell__count",
-            "value": 25188,
+            "value": 25083,
             "compare": "<="
         },
         {
             "field": "cts__timing__setup__ws",
-            "value": -300.0,
+            "value": 0.0,
             "compare": ">="
         },
         {
             "field": "cts__timing__setup__ws__prerepair",
-            "value": -300.0,
+            "value": 0.0,
             "compare": ">="
         },
         {
             "field": "cts__timing__setup__ws__postrepair",
-            "value": -300.0,
+            "value": 0.0,
             "compare": ">="
         },
         {
             "field": "globalroute__timing__clock__slack",
-            "value": -300.0,
+            "value": 0.0,
             "compare": ">="
         },
         {
             "field": "globalroute__timing__setup__ws",
-            "value": -300.0,
+            "value": 0.0,
             "compare": ">="
```

Comment:
why are there such large changes in the other metric rules?

Reply:
The previous behavior of the script allowed for `- period * padding` when the value was positive, now if the WS / slack is positive we only allow it to go to 0. This new behavior is to avoid unwanted degradations.